sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -72,6 +80,19 @@ class HFTransformersForecaster(BaseForecaster):
         When `fit_strategy` is set to "peft",
         this will be used to set up PEFT parameters for the model.
         See the `peft` documentation for details.
+    no_size1_batch: bool, default=True
```

Comment:
Why is this required?

Reply:
Probably some models are using BatchNormalization and in this case batch 1 can break or?

Reply:
yes, it's because of BatchNormalization, it happend in some of the CI test.